NEMO/templates/users/view_user.html

```diff
@@ -0,0 +1,70 @@
+{% extends 'base.html' %}
+{% load custom_tags_and_filters %}
+{% load mptt_tags %}
+{% block title %}View user{% endblock %}
+{% block content %}
+	{% block basic_info %}
+	<div class="panel panel-default">
+		<div class="panel-body">
+			<h3 id="basic_info_label">Basic Information</h3>
+			<ul>
+				<li><b>First name</b>: {{ user.first_name }}</li>
+				<li><b>Last name</b>: {{ user.last_name }}</li>
+				<li><b>Username</b>: {{ user.username }}</li>
+				<li><b>Email</b>: {{ user.email }}</li>
+				<li><b>Type</b>: {{ user.type }}</li>
+				<li><b>Date joined</b>: {{ user.date_joined }}</li>
+				<li><b>Last login</b>: {{ user.last_login }}</li>
+			</ul>
+		</div>
+	</div>
+	{% endblock %}
+	{% block projects %}
+	<div class="panel panel-default">
+		<div class="panel-body">
+			<h3 id="projects_label">Projects</h3>
+			{% if user.projects  %}
+				<ul>
+					{% for project in user.projects.all %}
+						<li>{{ project }}</li>
+					{% endfor %}
+				</ul>
+			{% else %}
+				<p>This user has no assigned projects.</p>
+			{% endif %}
+		</div>
+	</div>
+	{% endblock %}
+	{% block tool_qualifications %}
+	<div class="panel panel-default">
+		<div class="panel-body">
+			<h3 id="tool_qualifications_label">Tool Qualifications</h3>
+			{% if tool_qualifications  %}
+				<ul>
+					{% for tool_qualification in tool_qualifications %}
+						<li>{{ tool_qualification }}</li>
+					{% endfor %}
+				</ul>
+			{% else %}
+				<p>Not qualified for any tools.</p>
+			{% endif %}
+		</div>
+	</div>
+	{% endblock %}
+	{% block groups %}
+	<div class="panel panel-default">
+		<div class="panel-body">
+			<h3 id="groups_label">Groups</h3>
+			{% if groups  %}
+				<ul>
+					{% for group in groups %}
+						<li>{{ group }}</li>
+					{% endfor %}
+				</ul>
+			{% else %}
+				<p>Not assigned to any groups.</p>
+			{% endif %}
+		</div>
+	</div>
+	{% endblock %}
```

Comment:
I don't think we want users to see the authentication groups they are part of. that should stay hidden
```suggestion
```

Reply:
Should staff be able to see groups? We can add an is_staff check if staff would find that useful.

Reply:
yeah I think that would be fine

Reply:
Added the user.is_any_part_of_staff check